src/msg/DispatchQueue.cc

```diff
@@ -184,6 +184,14 @@ void DispatchQueue::entry()
 	case D_CONN_REFUSED:
 	  msgr->ms_deliver_handle_refused(qitem.get_connection());
 	  break;
+	case D_CONN_REAP:
+	  {
+	    auto con = qitem.get_connection();
+	    ldout(cct,10) << "DispatchQueue::entry reset con " << con
```

Comment:
shall we consider this a bug? and print this with a higher debug level, for instance, using `lderr(cct)`?

Reply:
it's triggers pretty easily..

Reply:
ahhh! maybe we can make it `lderr(cct)` after nautilus.